firmware/boron-ins-fsm/src/stateMachine.h

```diff
@@ -21,11 +21,14 @@
 
 //length of time between debug publishes
 #define DEBUG_PUBLISH_INTERVAL 1500  //ms
-#define SM_HEARTBEAT_INTERVAL 600000  //ms = 10 min
+#define SM_HEARTBEAT_INTERVAL 660000  //ms = 11 min
```

Comment:
Hmmm, can this go back to being 10 minutes?
I think the purpose behind the 11 minutes mentioned in the ClickUp Task is actually covered by the 15-minute `MSG_TRIGGER_SM_HEARTBEAT_THRESHOLD`.

Reply:
For sure, yeah